### ISSUE TITLE
useTransition : "Cannot destructure property 'ref' of 'props' as it is null."

### ISSUE THREAD

**Issue by Kais3rP** [pmndrs/react-spring#1235]
## 🐛 Bug Report

useTransition hook throws this exception both on a local project and on a stackblitz test.
"Cannot destructure property 'ref' of 'props' as it is null." - Local error
"Cannot read property 'ref' of null" - Stackblitz

## To Reproduce

I just copy-pasted the example of useTransition present in the docs.

## Expected behavior

I expected the docs example to work properly.
*Tested on react-spring v8.0.27 and it works fine

## Link to repro (highly encouraged)
*Not working on v9.0.0-rc.3
https://stackblitz.com/edit/react-2awmek
*Working on v8.0.27
https://stackblitz.com/edit/react-pc1vax

## Environment

- `react-spring` v9.0.0-rc.3
- `react` v17.0.1

**Comment by DanielBalog86** [comment]
You are using v9.0.0-rc3, which has breaking changes versus v8. You code will only work for v8.

Please read: https://github.com/pmndrs/react-spring.io/blob/v9/content/v9/breaking-changes.mdx

TLDR, transition is now a function instead of an array.

**Comment by Kais3rP** [comment]
Ok, thanks for the answer, you are right, I was mislead by this changelog: https://www.react-spring.io/log reporting the old way to use useTransition hook in the 9.0.0 changelog !
